.run/Generate Demo Baseline Profile.run.xml

```diff
@@ -0,0 +1,23 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Generate Demo Baseline Profile" type="GradleRunConfiguration" factoryName="Gradle">
```

Comment:
Can we add a comment pointing to documentation about baseline profiles, to aid in discoverability?